config/routes.rb

```diff
@@ -16,4 +16,6 @@
       resources :users, only: [:update]
     end
   end
+
+  resources :topics
```

Comment:
It is not under the V1 namespace because it is a shared resource with admin views